spec.html

```diff
@@ -155,4 +155,209 @@ <h1>Array.prototype.with ( _index_, _value_ )</h1>
             </emu-clause>
         </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-%typedarray%-intrinsic-object">
+      <h1>The %TypedArray% Intrinsic Object</h1>
+
+        <emu-clause id="sec-properties-of-the-%typedarrayprototype%-object">
+            <h1>Properties of the %TypedArray% Prototype Object</h1>
+
+            <emu-clause id="sec-%typedarray%.prototype.popped">
+                <h1>%TypedArray%.prototype.popped ( )</h1>
+
+                <p>When the *popped* method is called, the following steps are taken:</p>
+
+                <emu-alg>

```

Comment:
That's excellent, I think we might want to backport that style of spec text as suggested in #7

Reply:
I could do that as a separate PR? It may also help address #3 (working with array-like values which also don't have applicable mutator methods).

Reply:
yes definitely a separate pr!